### PR TITLE
Develop sofi

### PR DESCRIPTION
**Pull request by gabehal**

### CODE REVIEW

File: helperAPI.py [hunk 1]
```diff
@@ -505,6 +512,24 @@ def getDriver(DOCKER=False):
                 service=EdgeService(),
                 options=options,
             )
+            """
+             #! What I added
+            chrome_options = Options()
+            chrome_options.add_argument('--ignore-certificate-errors')
+            chrome_options.add_argument('--ignore-ssl-errors')
```
Comment (PZES) [inline helperAPI.py:519]: I dont think you should ignore cert and ssl errors when you are giving usernames and passwords to these sites

**Comment by PZES** [comment]
I think you should open a separate pf for the uc stuff as it affects everything not just sofi

File: helperAPI.py [hunk 2]
```diff
@@ -505,6 +512,24 @@ def getDriver(DOCKER=False):
...
+            chrome_options.add_argument('--disable-blink-features=AutomationControlled')
+            chrome_options.add_argument('--disable-notifications')
+            chrome_options.add_argument('--disable-popup-blocking')
+            chrome_options.add_argument('--disable-infobars')
+            chrome_options.add_argument('--disable-extensions')
+            chrome_options.add_argument("--no-sandbox")
+            chrome_options.add_argument("--disable-dev-shm-usage")
+            chrome_options.add_argument("--incognito")
```
Comment (PZES) [inline helperAPI.py:527]: is there a specific reason to use incognito

Comment (gabehal) [reply helperAPI.py:527]: I got tired of the save passwords popups at every login

**Comment by gabehal** [comment]
> I think you should open a separate pr for the uc stuff as it affects everything not just sofi

Fair enough, the use of uc is what ended up working, otherwise sofi would continuously ask to verify you're not a bot.

**Comment by NelsonDane** [comment]
Is this replaced by https://github.com/NelsonDane/auto-rsa/pull/314 @gabehal ?

**Pull request closed without merging**